guacamole-ext/src/main/java/org/apache/guacamole/net/auth/simple/SimpleConnection.java

```diff
@@ -157,6 +162,43 @@ public SimpleConnection(String name, String identifier,
         this.fullConfig = config;
         this.interpretTokens = interpretTokens;
 
+    }
+    
+        /**
+     * Creates a new SimpleConnection having the given identifier,
+     * GuacamoleConfiguration, and GuacamoleProxyConfiguration. Parameter tokens
+     * will be interpreted if explicitly requested.
+     *
+     * @param name
+     *     The name to associate with this connection.
+     *
+     * @param identifier
+     *     The identifier to associate with this connection.
+     *
+     * @param config
+     *     The configuration describing how to connect to this connection.
+     *
+     * @param interpretTokens
+     *     Whether parameter tokens in the underlying GuacamoleConfiguration
+     *     should be automatically applied upon connecting. If false, parameter
+     *     tokens will not be interpreted at all.
+     * 
+     * @param proxyConfig
+     *     The Guacamole proxy configuration used to tell the client how to
+     *     connect to guacd.
```

Comment:
> The Guacamole proxy configuration used **to tell the client** how to connect to guacd.

I think the wording here is a bit confusing, since the code invoking this constructor _is_ the client. I suggest rewording to something like: "A GuacamoleProxyConfiguration object describing how the connection to guacd should be established."

(I'm partly copy/pasting from https://guacamole.apache.org/doc/1.3.0/guacamole-ext/org/apache/guacamole/net/auth/GuacamoleProxyConfiguration.html)

Reply:
OKay, makes sense.